openvdb/unittest/TestAttributeSet.cc

```diff
@@ -581,13 +581,17 @@ TestAttributeSet::testAttributeSet()
         attrSet.appendAttribute("vector", AttributeVec3s::attributeType(), /*stride=*/1);
         attrSet.appendAttribute("vector3", AttributeVec3s::attributeType(), /*stride=*/3);
         attrSet.appendAttribute("bool100", AttributeB::attributeType(), /*stride=*/100);
+        attrSet.appendAttribute("boolNonUniform", AttributeB::attributeType(), /*size=*/100, false);
+        attrSet.appendAttribute("intNonUniform", AttributeI::attributeType(), /*size=*/300, false);
 
         CPPUNIT_ASSERT_EQUAL(std::string("float"), attrSet.getConst("float1")->type().first);
         CPPUNIT_ASSERT_EQUAL(std::string("int32"), attrSet.getConst("int1")->type().first);
         CPPUNIT_ASSERT_EQUAL(std::string("float"), attrSet.getConst("float3")->type().first);
         CPPUNIT_ASSERT_EQUAL(std::string("vec3s"), attrSet.getConst("vector")->type().first);
         CPPUNIT_ASSERT_EQUAL(std::string("vec3s"), attrSet.getConst("vector3")->type().first);
         CPPUNIT_ASSERT_EQUAL(std::string("bool"), attrSet.getConst("bool100")->type().first);
+        CPPUNIT_ASSERT_EQUAL(std::string("bool"), attrSet.getConst("boolNonUniform")->type().first);
```

Comment:
I think I'd refer to this as dynamic rather than non uniform to avoid confusion, as we usually refer to arrays as uniform in the context of uniform value compression.

Reply:
done